include/spirv-tools/optimizer.hpp

```diff
@@ -393,6 +393,13 @@ Optimizer::PassToken CreateAggressiveDCEPass();
 // The pass remaps result ids to a compact and gapless range starting from %1.
 Optimizer::PassToken CreateCompactIdsPass();
 
+// Creates a CFG cleanup pass.
+// This pass removes cruft from the control flow graph. It currently includes
+// the following functionality:
```

Comment:
The pass only operates on functions reachable from some entry point.  Should say so.

Reply:
Done.